src/frontend/Komponenter/Uttrekk/UttrekkArbeidssøker.tsx

```diff
@@ -0,0 +1,229 @@
+import { Checkbox } from 'nav-frontend-skjema';
+
+const UttrekkArbeidssøkerContent = styled.div`
+    padding: 1rem;
+`;
+
+const StyledTable = styled.table`
+    width: 60%;
+    padding: 2rem;
+    margin-left: 1rem;
+`;
+
+const StyledPagination = styled(Pagination)`
+    padding: 1rem;
+`;
+
+interface UttrekkArbeidssøkere {
+    årMåned: string;
+    side: number;
+    antallTotalt: number;
+    antallKontrollert: number;
+    arbeidssøkere: UttrekkArbeidsssøker[];
+}
+
+interface UttrekkArbeidsssøker {
+    id: string;
+    fagsakId: string;
+    behandlingIdForVedtak: string;
+    kontrollert: boolean;
+    kontrollertTid?: string;
+    kontrollertAv?: string;
+}
+
+const URL_ARBEIDSSØKER = '/familie-ef-sak/api/uttrekk/arbeidssoker';
+
+const settArbeidssøkereTilKontrollert = (
+    prevState: RessursSuksess<UttrekkArbeidssøkere>,
+    id: string,
+    kontrollert: boolean
+) => {
+    return {
+        ...prevState,
+        data: {
+            ...prevState.data,
+            arbeidssøkere: prevState.data.arbeidssøkere.map((arbeidssøker) =>
+                arbeidssøker.id === id ? { ...arbeidssøker, kontrollert } : arbeidssøker
+            ),
+        },
+    };
+};
+
+const ANTALL_PER_SIDE = 20;
+const QUERY_PARAM_SIDE = 'side';
+const QUERY_PARAM_KONTROLLERTE = 'kontrollerte';
+
+const UttrekkArbeidssøker: React.FC = () => {
+    const query = useQueryParams();
+    const history = useHistory();
+
+    const side = query.get(QUERY_PARAM_SIDE) || 1;
+    const visKontrollerte = query.get(QUERY_PARAM_KONTROLLERTE) === 'true';
+
+    const [arbeidssøkere, settArbeidssøkere] = useState<Ressurs<UttrekkArbeidssøkere>>(
+        byggTomRessurs()
+    );
+    const [feilmelding, settFeilmelding] = useState<string>();
+    const { axiosRequest, gåTilUrl } = useApp();
+
+    const hentUttrekk = useCallback(
+        (side, visKontrollerte: boolean) => {
+            return axiosRequest<UttrekkArbeidssøkere, null>({
+                method: 'GET',
+                url: `${URL_ARBEIDSSØKER}?side=${side}${
+                    visKontrollerte ? '&visKontrollerte=true' : ''
+                }`,
+            }).then((respons: RessursSuksess<UttrekkArbeidssøkere> | RessursFeilet) => {
+                if (respons.status === RessursStatus.SUKSESS) {
+                    settArbeidssøkere(respons);
+                } else {
+                    settFeilmelding(respons.frontendFeilmelding);
+                }
+            });
+        },
+        [axiosRequest]
+    );
+
+    const settKontrollert = useCallback(
+        (id: string, kontrollert: boolean): void => {
+            const kontrollertQuery = kontrollert ? '?kontrollert=false' : '';
+            axiosRequest<UttrekkArbeidssøkere, null>({
+                method: 'POST',
+                url: `${URL_ARBEIDSSØKER}/${id}/kontrollert${kontrollertQuery}`,
+            }).then((respons: RessursSuksess<UttrekkArbeidssøkere> | RessursFeilet) => {
+                settFeilmelding(undefined);
```

Comment:
Flytt denne opp til 107 / 108 for å fjerne feilmeldingen FØR vi utfører kallet. Hvis ikke kan det oppleves som at den har dukket opp på nytt dersom http-kallet tar litt tid.